Practice2/src/partials/top-section.html

```diff
@@ -0,0 +1,39 @@
+<header class="page-top">
+  <div>
+    <picture>
```

Comment:
You are using the picture tag wrong. Please check source min-width careful

Reply:
Fixed